save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/demo/CpgView.kt

```diff
@@ -52,6 +62,8 @@ val cpgView: VFC = VFC {
                             loadingHandler = ::loadingHandler,
                         )
 
+                        setErrorMessage(response.unpackMessage())
```

Comment:
Does spring provide error messages in `body["message"]` (AFAIR it needs special configuration for that)? Please test that error messages are user-friendly enough (if not, leave `TODO` here)

Reply:
Add error transmission from the backend should @nulls

Reply:
Oh ok